.drone.yml

```
@@ -26,18 +26,47 @@ steps:
         - push
         - pull_request
 
-  - name: build_image
+  - name: build_tag_push_ecr
     pull: if-not-exists
-    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+    image: plugins/ecr
     environment:
-      DOCKER_HOST: tcp://docker:2375
+      AWS_REGION: eu-west-2
+    settings:
+      access_key:
+        from_secret: ecr_timecard_restapi_access_key_id
+      secret_key:
+        from_secret: ecr_timecard_restapi_secret_access_key
+      repo: callisto/timecard-restapi
+      registry: 340268328991.dkr.ecr.eu-west-2.amazonaws.com
+      tags:
+      - latest
+    when:
+      event:
+        - push
+      branch:
+        - main
+
+---
+
+kind: pipeline
+type: kubernetes
+name: callisto-timecard-database
```

```suggestion
name: callisto-timecard-database-schema
```
I don’t think we’ve done a great job of sharing the broader plan yet but I _think_ the database itself is going to be shared between all of our services, and it’s the database _schema_ that’s within this pipeline’s remit